content/docs/use-cases/model-registry.md

```diff
@@ -2,68 +2,63 @@
 
 A **model registry** is a tool to catalog ML models and their versions. Models
 from your data science projects can be discovered, tested, shared, deployed, and
-audited from there. [DVC](/doc), [GTO], and [MLEM] enable these capabilities on
-top of Git, so you can stick to an existing software engineering stack. No more
+audited from there. [DVC], [GTO], and [MLEM] enable these capabilities on top of
+Git, so you can stick to an existing software engineering stack. No more
 division between ML engineering and operations!
 
+In addition, [Studio model registry] enables your teams to **collaborate** on
+projects with greater efficiency. It provides a unified platform to organize,
+discover, version, manage model stages (e.g. dev, shadow, prod) and track their
+lineage using these open-source tools.
+
 ![](/img/ml_model_registry.jpg) _MLOps from modeling to production_
 
 [gto]: https://mlem.ai/doc/gto
 [mlem]: https://mlem.ai/
 
-ML model registries give your team key capabilities:
-
-- Collect and organize model [versions] from different sources effectively,
-  preserving their data provenance and lineage information.
-- Share metadata including [metrics and plots][mp] to help use and evaluate
-  models.
-- A standard interface to access all your ML artifacts, from early-stage
-  [experiments] to production-ready models.
-- Deploy specific models on different environments (dev, shadow, prod, etc.)
-  without touching the applications that consume them.
-- For security, control who can manage models, and audit their usage trails.
-
-[versions]: /doc/use-cases/versioning-data-and-models
-[mp]: /doc/start/data-management/metrics-parameters-plots
-[experiments]: /doc/user-guide/experiment-management
-
-Many of these benefits are built into DVC: Your [modeling process] and
-[performance data][mp] become **codified** in Git-based <abbr>DVC
-repositories</abbr>, making it possible to reproduce and manage models with
-standard Git workflows (along with code). Large model files are stored
-separately and efficiently, and can be pushed to [remote storage] -- a scalable
-access point for [sharing].
+ML model registry enable end-to-end workflows:
 
-<admon type="info">
+- **Add/Log your model using dvclive**: Start by logging your model's
```

Comment:
```suggestion
- **Add/Log your model**: Start by logging your model's
```

Minor, but again I wouldn't overly focus on the individual tools here. This doc does not need to explain how it works or even how to get started using it.

Reply:
Actually, I would drop all the mentions of any tools other than Studio in this doc. We could mention somewhere that all of these operations are supported by open-source tools and that you can use the model registry without Studio, but that Studio is the easiest way to use it and has benefits like interfaces to compare, register, and promote models across all your team's projects.